client/lib/executable_deploy_item_ext.rs

```diff
@@ -19,17 +19,17 @@ pub trait ExecutableDeployItemExt {
         args: RuntimeArgs,
     ) -> Result<ExecutableDeployItem>;
 
-    /// Creates an `ExecutableDeployItem::StoredVersionedContractByHash`.
-    fn new_stored_versioned_contract_by_hash(
-        hash: ContractPackageHash,
+    /// Creates an `ExecutableDeployItem::StoredVersionedContractByName`.
+    fn new_stored_versioned_contract_by_name(
+        name: String,
         version: Option<u32>,
         entry_point: String,
         args: RuntimeArgs,
     ) -> Result<ExecutableDeployItem>;
 
-    /// Creates an `ExecutableDeployItem::StoredVersionedContractByName`.
-    fn new_stored_versioned_contract_by_name(
-        name: String,
+    /// Creates an `ExecutableDeployItem::StoredVersionedContractByHash`.
+    fn new_stored_versioned_contract_by_hash(
+        hash: ContractPackageHash,
```

Comment:
Did you just flip those two (`_by_hash` and `_by_name`)?

Reply:
Yeah, the ordering was different from trait to implementation.